ArduCopter/mode_auto.cpp

```diff
@@ -1647,8 +1647,10 @@ void ModeAuto::do_planck_wingman(const AP_Mission::Mission_Command& cmd)
       Vector3f(
         cmd.content.planck_wingman.x,
         cmd.content.planck_wingman.y,
-        (float)cmd.content.planck_wingman.z_cm/100.
-      )
+        (float)cmd.content.planck_wingman.z_cm/100.),
+      false,
```

Comment:
Indentation incorrect

Reply:
why is it incorrect? the "false" and "copter..." lines line up with the "Vector.3f(..." line, the others are indented because they are arguments to Vector.3f()

Reply:
Got it, my bad.